sites/partners/src/components/listings/PaperListingForm/UnitGroupForm.tsx

```diff
@@ -370,6 +377,79 @@ const UnitGroupForm = ({
                   />
                 </Grid.Cell>
               </Grid.Row>
+              {isNonRegulated && (
+                <>
+                  <Grid.Row columns={3}>
+                    <Grid.Cell>
+                      <FieldGroup
+                        name="rentType"
+                        type="radio"
+                        register={register}
+                        groupLabel={t("listings.unitGroup.rentType")}
+                        fields={[
+                          {
+                            label: t("listings.unitGroup.fixedRent"),
+                            value: RentTypeEnum.fixedRent,
+                            id: "rentTypeFixed",
+                          },
+                          {
+                            label: t("listings.unitGroup.rentRange"),
+                            value: RentTypeEnum.rentRange,
+                            id: "rentTypeRange",
+                          },
+                        ]}
+                      />
+                    </Grid.Cell>
+                  </Grid.Row>
+                  {rentType === RentTypeEnum.fixedRent && (
+                    <Grid.Row columns={3}>
+                      <Grid.Cell>
+                        <Field
+                          id="monthlyRent"
+                          name="monthlyRent"
+                          label={t("listings.unit.monthlyRent")}
+                          register={register}
+                          placeholder="0.00"
+                          type="number"
+                          prepend="$"
+                        />
+                      </Grid.Cell>
+                    </Grid.Row>
+                  )}
+                  {rentType === RentTypeEnum.rentRange && (
+                    <Grid.Row columns={3}>
+                      <Grid.Cell>
+                        <Field
```

Comment:
issue: The error message doesn't disappear here after entering a valid "Monthly Rent To". 

Steps to reproduce:
1. On a new form enter a "Monthly Rent From" value, the error immediately appears
2. Enter a "Monthly Rent To" value greater than the from value. An error appears for this field as you type until the number is greater than the "from".
3. The "Monthly Rent From" value doesn't disappear until you do another action such as "Save & Exit" or change the value of "Monthly Rent From"

<img width="678" height="164" alt="Image" src="https://github.com/user-attachments/assets/37b40544-914e-4cf0-adc2-5d0a5d00bc84" />